### DISCUSSION TITLE
Does this extension not recognize tsconfig.json?

### DISCUSSION THREAD

**Discussion by ghost** [vuejs/language-tools#1540] [Q&A] [answered]
I configured tsconfig.json trying to use hasOwn added in es2022 but the error doesn't go away.

<img width="671" alt="スクリーンショット 2022-07-08 5 54 38" src="https://user-images.githubusercontent.com/68766032/177869637-33509b85-2a58-4be9-9c21-58bc6d09e184.png">

I noticed that the Volar extension is `No tsconfig`. 
I'm wondering if this is the cause.
Does this extension recognize tsconfig.json?
Also, if this extension doesn't recognize tsconfig.json, is there a workaround for this issue?

<img width="180" alt="スクリーンショット 2022-07-08 5 54 27" src="https://user-images.githubusercontent.com/68766032/177869627-96537d76-4cc2-421a-8ee2-f7573a784412.png">

**Comment by johnsoncodehk** [comment, marked as answer]
Do you have add file path to tsconfig's `include` option? See https://github.com/johnsoncodehk/volar-starter/blob/9ce7ac0c2a007aaad3d0819edd84c2cd179aa843/tsconfig.json#L19

**Comment by ghost** [comment]
@johnsoncodehk 
Thank you for reply
I had set the include option, but when I checked it again, it seemed that another extension was in the way and was tampered with to another path.

I'm sorry to bother you with my silly mistake.

**Reply by johnsoncodehk** [reply]
Don't say that, it's fine. :)
